### PR TITLE
TNO-1652: Show Results of Searches

### PR DESCRIPTION
**Pull request by jdtoombs**
Please note that the app is in a bit of styling limbo as some components have been updated with the new MMI style, while others have not.

Also one more thing to note, I have to talk to Bobbi about something in regards to updating a search. Going to add that in once I gather that information. **_Leaving in draft; however, will include these changes in a separate smaller commit to keep the review easy,._** 

- user can now navigate from my searches to search results
- styling updated
- a few other styling and scrollbar fixes

image

adv-search-link

### CODE REVIEW

File: app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx [hunk 1]
```diff
@@ -101,6 +110,21 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({
     [advancedSearch, constants?.frontPageId],
   );
 
+  const saveSearch = React.useCallback(async () => {
+    const data = await addFilter({
```
Comment (Fosol) [inline app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx:114]: Looks like a unhandled promise to me.  Testing with the developer console normally shows an error for this stuff.

File: app/subscriber/src/css/_colors.scss
Comment (Fosol) [inline app/subscriber/src/css/_colors.scss:1]: We need a proper theme setup with relevant colours and sizes.  This has been a mess for too long.  Need to make a story to do this.

File: app/subscriber/src/features/my-searches/MySearches.tsx [hunk 2]
```diff
@@ -80,6 +101,7 @@ export const MySearches = () => {
         onConfirm={() => {
           try {
             if (!!active) {
+              alert('active');
```
Comment (Fosol) [inline app/subscriber/src/features/my-searches/MySearches.tsx:104]: What does this do?

**Comment by jdtoombs** [comment]
@Fosol - I think I am going to open another small ticket for updating an existing search as it is not quite ready to be worked on yet. Need more clarification. Going to mark this one as ready for review

**Review by Fosol — APPROVED** [review]

**Pull request merged**